sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -754,6 +754,47 @@ class HiveDDLSuite
     }
   }
 
+  test("Insert overwrite Hive table should output correct schema") {
+    withTable("tbl", "tbl2") {
+      withView("view1") {
+        spark.sql("CREATE TABLE tbl(id long)")
```

Comment:
please run this test within `withSQLConf(HiveUtils.CONVERT_METASTORE_PARQUET -> false)`

Reply:
I am not familiar with Hive. But as I look at the debug message of this logical plan, the top level is `InsertIntoHiveTable `default`.`tbl2`, org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe, true, false, [ID]`. It should not be related to this configuration, right?